docs/product/insights/mobile-vitals/screen-loads.mdx

```diff
@@ -17,13 +17,13 @@ Sentry tracks TTID automatically, but [TTFD](/product/insights/mobile-vitals/#ti
 **For Android:**
 
 - `>=5.0` for automatic activity transactions
-- `>=6.10.0` for TTID
-- `>=6.17.0` for TTFD, (released March, 2023)
+- `>=6.10.0` for [TTID](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-initial-display)
+- `>=6.17.0` for [TTFD](/platforms/android/tracing/instrumentation/automatic-instrumentation/#time-to-full-display), (released March, 2023)
```

Comment:
Just in case, flagging that the linked page says
> (New in version 6.14.0)

but this page has `>=6.17.0`. Not related to this change though, which LGTM 👍

Reply:
@mjq ah, that's confusing lol thanks for catching that. I looked around and it's because 6.14.0 introduced the `ui.load.full_display` _span_ whereas 6.17.0 adds the measurement used for full display. I'll merge this as is and figure out which is the minimum version necessary, the goal of this PR was to link to a code snippet in the docs